webapp/packages/plugin-authentication-administration/src/Administration/Users/Teams/Options/TeamOptionsTabService.ts

```diff
@@ -102,17 +103,24 @@ export class TeamOptionsTabService extends Bootstrap {
 
   private async save({ state }: ITeamFormSubmitData, contexts: IExecutionContextProvider<ITeamFormSubmitData>) {
     const status = contexts.getContext(this.teamFormService.configurationStatusContext);
-    const config = contexts.getContext(teamContext);
+    const { metaParameters, ...config } = contexts.getContext(teamContext);
 
     const create = state.mode === 'create';
 
     try {
       if (create) {
-        const team = await this.teamResource.createTeam(config);
+        const [team] = await Promise.all([
+          this.teamResource.createTeam(config),
+          this.teamsMetaParametersResource.setMetaParameters(config.teamId, metaParameters),
+        ]);
+
         status.info('administration_teams_team_info_created');
         status.info(team.teamId);
       } else {
-        const team = await this.teamResource.updateTeam(config);
+        const [team] = await Promise.all([
```

Comment:
I think here we dont want to run these mutations in parallel as we dont want to set parameters if we failed to update the team

Reply:
good point

Reply:
nice one! thanks